src/test/java/redis/clients/jedis/commands/unified/UnifiedJedisCommandsTestBase.java

```diff
@@ -1,11 +1,33 @@
 package redis.clients.jedis.commands.unified;
 
+import java.util.Arrays;
+import java.util.Collection;
+
+import org.junit.runners.Parameterized;
+import redis.clients.jedis.RedisProtocol;
 import redis.clients.jedis.UnifiedJedis;
 
 public abstract class UnifiedJedisCommandsTestBase {
 
-  protected static UnifiedJedis jedis;
+  /**
+   * Input data for parameterized tests. In principle all subclasses of this
+   * class should be parameterized tests, to run with the default protocol,
+   * with RESP2 and with RESP3.
+   */
+  @Parameterized.Parameters
```

Comment:
Is there a way to add this to a superclass of both 
src/test/java/redis/clients/jedis/commands/unified/UnifiedJedisCommandsTestBase.java
and 
src/test/java/redis/clients/jedis/commands/jedis/JedisCommandsTestBase.java
so that we do not have duplicate code?

Reply:
To not end up with an even more intricate inheritance hierarchy, I tried to rather compose this by calling a shared static method. Would that work?

Reply:
Yes, sure, that's fine.
It would only make sense to have two different implementations if they actually provide a different set for some reason.